tests/lib/Birthday.test.ts

```diff
@@ -276,6 +276,18 @@ describe('Birthday', () => {
 			test('GIVEN day after now THEN returns 19', () => {
 				expect(getAge({ year: 2000, month: Month.May, day: 5 })).toBe(19);
 			});
+
+			test('GIVEN yesterday THEN returns 20', () => {
+				expect(getAge({ year: 2000, month: Month.April, day: 17 })).toBe(20);
+			});
+
+			test('GIVEN today THEN returns 20', () => {
+				expect(getAge({ year: 2000, month: Month.April, day: 18 })).toBe(20);
```

Comment:
April 18th 2000 -> April 17th 2020 should be 19
April 18th 2000 -> April 18th 2020 should be 20
April 18th 2000 -> April 19th 2020 should be 20

So the first and last test cases here are incorrect

Reply:
Not actually, if i got it right, what we are changing here is the date of birth, not the current date. So:

Date of birth -> Current Date
April 17th 2000 -> April 18th 2020 should be 20 (birthday was yesterday)
April 18th 2000 -> April 18th 2020 should be 20 (birthday is today)
April 19th 2000 -> April 18th 2020 should be 19 (birthday is tomorrow)

Reply:
I can update the tests descriptions so it becomes more explicity (i also got a bit confused while writting it)

Reply:
I'm a bit confused right now as well. What I can certainly see is that where we use `getAge` in `upcoming-birthdays` it does return the correct result right now. 

With the unchanged code it should return 26 for me. That is `February 21st 1995` -> `February 3rd 2022`. Then in `upcoming-birthday` we do `+ 1` on the age and the embed correctly shows 27 for me.

![image](https://user-images.githubusercontent.com/4019718/152416958-edd086c1-7c2d-4a2e-9ce9-45fce799d3d2.png)

Reply:
https://github.com/skyra-project/skyra/blob/54a95578d7eb5d18b4417c9c173bda5ac318842e/src/commands/Birthday/upcoming-birthdays.ts#L44

So, if im not wrong, in this `getAge`, you pass my birthdate (mine for exemple, 02/03/1998) and the optional date, which i believe is the date that the next task is schedule (mine for exemple, 02/03/2023).

With the unchanged `getAge`, if you are on the same date as you birthdate, it will still return your old age, so in the exemple above, it would return 24, then you increment +1 which returns 25.

The problem when the bot announces someone birthday, because we are in the same date as my birthdate, the `getAge` returns my old age, and this value is not incremented like in the `upcoming-birthdays`.

Reply:
I update the method with a different getAge algorithm, if you could test it i would appreciate.

Reply:
I hope that using getUTC... might fix any other issues